infor-design/enterprise
# 3596 - Add dropdown positional fixes

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Fixes a few positional issues of the dropdown, when validated on a modal, flowing upwards and in RTL.

**Related github/jira issue (required)**:
Fixes #2068
Fixes #3596

**Steps necessary to review your pull request (required)**:
**Test One**
- go to http://localhost:4000/components/modal/example-validation.html
- Tab all fields
- Open the dropdown field
- It should not overlap the Type label

**Test Two**
- go to http://localhost:4000/components/timepicker/test-24-hour?locale=he-IL
- open the time picker and then the dropdowns
- position should be correct

**Test Three**
- go to http://localhost:4000/components/field-filter/example-short.html?theme=uplift&variant=contrast
- open the time picker and then the dropdowns
- position should be correct

**Test Four**
- go to http://localhost:4000/components/datepicker/example-timeformat.html
- open the second picker and then the dropdowns
- position should be correct

**Included in this Pull Request**:
